dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/GetOrderStatusInfoByIdWithApiKeySpec.scala

```diff
@@ -0,0 +1,166 @@
+package com.wavesplatform.it.matcher.api.http
+
+import com.softwaremill.sttp.StatusCodes
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.http.entities.HttpOrderBookHistoryItem
+import com.wavesplatform.dex.domain.account.KeyPair.toAddress
+import com.wavesplatform.dex.domain.bytes.codec.Base58
+import com.wavesplatform.dex.domain.order.Order
+import com.wavesplatform.dex.domain.order.OrderType.{BUY, SELL}
+import com.wavesplatform.dex.it.api.RawHttpChecks
+import com.wavesplatform.dex.it.docker.apiKey
+import com.wavesplatform.dex.model.{AcceptedOrderType, OrderStatus}
+import com.wavesplatform.it.MatcherSuiteBase
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class GetOrderStatusInfoByIdWithApiKeySpec extends MatcherSuiteBase with TableDrivenPropertyChecks with RawHttpChecks {
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(
+    s"""waves.dex {
+       |  price-assets = [ "$BtcId", "$UsdId", "WAVES" ]
+       |}""".stripMargin
+  )
+
+  val order = mkOrder(alice, wavesUsdPair, BUY, 10.waves, 2.usd)
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueBtcTx, IssueUsdTx)
+    dex1.start()
+  }
+
+  "GET /matcher/orders/{address}/{orderId} " - {
+
+    def httpOrderInfo(o: Order, s: OrderStatus, f: Long = 0.003.waves, awp: Long = 0, tepa: Long = 0): HttpOrderBookHistoryItem =
```

Comment:
What is `awp`? ![awp](https://csgo-academy.net/wp-content/uploads/2019/02/CSGO_AWP-728x409.jpg)
This? :D

Please give it better name and `tepa` too

Reply:
renamed awp to b46 ))